modules/expo-bluesky-translate/ios/ExpoBlueskyTranslateView.swift

```diff
@@ -2,36 +2,21 @@ import ExpoModulesCore
 import Foundation
 import SwiftUI
 
-class TranslateViewProps: ObservableObject {
-  @Published var text: String = ""
-  @Published var children: [UIView]?
-  @Published var onClose: EventDispatcher
-  @Published var isPresented: Bool = false {
-    didSet {
-      if !isPresented {
-        self.onClose()
-      }
-    }
-  }
-
-  init(onClose: EventDispatcher) {
-    self.onClose = onClose
-  }
+class TranslateViewState: ObservableObject {
+  static var shared = TranslateViewState()
```

Comment:
If we were not targeting iOS 13.4, we could just use `@StateObject` from the top and not worry about making this a singleton. In any case, this results in the same thing.